core/unit_test/TestExecutionSpace.hpp

```diff
@@ -57,7 +57,7 @@ struct CheckClassWithExecutionSpaceAsDataMemberIsCopyable {
     auto copy = *this;
     // not actually doing anything useful, mostly checking that
     // ExecutionSpace::in_parallel() is callalable
```

Comment:
There's a typo `callalable` right here you could fix while you're here
```suggestion
    // ExecutionSpace::in_parallel() is callable
```